js/marquee.js

```diff
@@ -0,0 +1,131 @@
+class StockBarData {
```

Comment:
Exepct using the class for storing the symbol and price, you were supposed to include all the marquee functionalityies inside.